Common.xml

```diff
@@ -3519,13 +3519,13 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
       </Item>
       
       <Item ID="6049">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
-        <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Name>Measurement Quality Level</Name>     
+        <Operations>R</Operations>     
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
```

Comment:
I think this is actually better without unit since it’s not strictly percentage. And we don’t have yet unit ID for 0-100% (it’s coming though)

Reply:
We recommend removing the unit.  9/19/19

Reply:
The unit was already removed on Aug 2 commit.